microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,35 +93,80 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 
 
 <!-- ====================================================================== -->
-## Before you begin
+## Enabling the Update REST API at Partner Center
+<!-- todo: apply tabsets, generally per section/heading -->
+
+To use the Microsoft Edge Add-ons API, you first need to enable the API for your project in the Microsoft Partner Center by creating API credentials, as follows:
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+##### [v1.1](#tab/v1-1)
 
-1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
+1. At your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), sign in to the account that you used to publish an add-on.
 
 1. Under the **Microsoft Edge** program, select **Publish API**.
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **API key**
+   * **Expiry date**
+
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)<!-- todo: add/use new png showing v1.1.  What's the title of the page that contains the UI control to select v1 vs v1.1?  What's the label of the UI control?  Is the UI control an Option button, or a Button? -->
+
+1. Write down the following:
+   * The **Client ID**.
+   * The **API key**
+
+   You'll use these values in the next step, to get an access token.<!-- todo: applies to 1.1? -->
+
+The ClientId and API Keys are automatically re-generated, by backend services.<!-- todo: every time you use any endpoint of the REST API?  what action causes re-generation of ClientId and API Keys? -->
```

Comment:
not clear why / where we should mention that ClientID & API key are auto gen'd.  State what will be done w/ them, which step(s)?  Why write down?

Reply:
closing Outdated comments.  added todo notes in .md; will create fresh Comment at each todo note